recipes/cuda-nvtx/bld.bat

```diff
@@ -0,0 +1,3 @@
+if not exist %PREFIX% mkdir %PREFIX%
+
+move include\nvtx3 %LIBRARY_INC%
```

Comment:
This is missing libraries.

Not sure if we also want to include bin for completeness. IMO we've been doing that in all packages, even those without binaries, so we may as well stay consistent, but it's not a big deal.

Reply:
There are no libraries in the Windows source. 

Which components have bin when it doesn't exist? This script would fail in that case.

Reply:
Hmm [libcurand](https://github.com/conda-forge/libcurand-feedstock/blob/main/recipe/bld.bat) and [libcublas](https://github.com/conda-forge/libcublas-feedstock/blob/main/recipe/bld.bat), for instance. Maybe this is a case where we haven't been careful enough in our Windows scripts since we're not building Windows packages atm?